epowcore/pandapower/pandapower_model.py

```diff
@@ -0,0 +1,344 @@
+            power_station_trafo=nan,
+            in_service=True,
+            slack_weight=0.0,
+        )
+        return True
+
+    def create_line_from_gdf_tline(self, core_model: CoreModel, tline: TLine):
+        """Create a pandapower line in the network equivalent to a gdf transmission line"""
+        # Get the neigbours of the transmission line to know what it connects to
+        from_bus = core_model.get_neighbors(component=tline, follow_links=True, connector="A")[0]
+        to_bus = core_model.get_neighbors(component=tline, follow_links=True, connector="B")[0]
+        # Conversion fails if one of the buses isn't found
+        if from_bus is None or to_bus is None:
+            print("Conversion of " + tline.name + " failed")
+            return False
+        network_frequency = core_model.base_frequency
+        # Create line in pandapower network
+        pandapower.create_line_from_parameters(
+            net=self.network,
+            name=tline.name,
+            index=tline.uid,
+            geodata=tline.coords,
+            from_bus=from_bus.uid,
+            to_bus=to_bus.uid,
+            length_km=tline.length,
+            r_ohm_per_km=tline.r1,
+            x_ohm_per_km=tline.x1,
+            c_nf_per_km=(tline.b1 * 1e3) / (2 * pi * network_frequency),
+            max_i_ka=0.5,
+            type=None,
+            in_service=True,
+            df=1.0,
+            parallel=tline.parallel_lines,
+            g_us_per_km=0.0,
+            max_loading_percent=nan,
+            alpha=tline.get_default(attr="alpha", platform=self.platform),
+            temperature_degree_celsius=tline.get_default(
+                attr="temperature_degree_celsius", platform=self.platform
+            ),
+            r0_ohm_per_km=tline.r0_fb(platform=self.platform),
+            x0_ohm_per_km=tline.x0_fb(platform=self.platform),
+            c0_nf_per_km=nan,
+            g0_us_per_km=0,
+            endtemp_degree=nan,
+        )
+        return True
+
+    def create_ward_from_gdf_ward(self, core_model: CoreModel, ward: Ward):
+        """Creates a ward in the pandapower network equivalent to a given
+        ward from the gdf.
+        """
+        ward_bus = get_connected_bus(core_model.graph, ward, max_depth=1)
+        # If there was no ward_bus found the function failed and terminates
+        if ward_bus is None:
+            return False
+        # Create ward in pandapower network
+        pandapower.create_ward(
+            net=self.network,
+            name=ward.name,
+            index=ward.uid,
+            bus=ward_bus,
+            ps_mw=-ward.p_gen + ward.p_load,
+            qs_mvar=-ward.q_gen + ward.q_load,
+            pz_mw=ward.p_zload,
+            qz_mvar=ward.q_zload,
+            in_service=True,
+        )
+        return True
+
+    def create_shunt_from_gdf_shunt(self, core_model: CoreModel, shunt: Shunt):
+        """Creates a shunt in the pandapower network equivalent to a given
```

Comment:
Stay with a common style: "Create a shunt..."
Also applies to the previous and next method.